### PR TITLE
Fix the bug of pendingKeys after local clear

### PR DESCRIPTION
**Pull request by clarenceli-msft**
[AB#4609](https://dev.azure.com/fluidframework/internal/_workitems/edit/4609)

It seems necessary to eliminate the contents in `pendingKeys` after a local clear operation as there will be no requirement to ack them subsequently.

### CODE REVIEW

File: packages/dds/map/src/mapKernel.ts [hunk 1]
```diff
@@ -342,6 +342,9 @@ export class MapKernel {
 		// Clear the data locally first.
 		this.clearCore(true);
 
+		// Clear the pendingKeys immediately, the local unack'd operations are aborted
+		this.pendingKeys.clear();
```
Comment (Abe27342) [inline packages/dds/map/src/mapKernel.ts:346]: seems reasonable, but I think you also need to update logic in resubmit--specifically I think we're now in danger of hitting 0x2ff.

**Review by Abe27342 — COMMENTED** [review]
It's a bit hard to judge this in isolation, as I don't think our map coverage is particularly good and there are other issues that make fuzz tests fail. Seeing as this change is very small, I'd kind of prefer you investigate the other skipped fuzz test and fix them together, along with a checkin that enables `clear` for the fuzz tests.

**Comment by Abe27342** [comment]
> It's a bit hard to judge this in isolation, as I don't think our map coverage is particularly good and there are other issues that make fuzz tests fail. Seeing as this change is very small, I'd kind of prefer you investigate the other skipped fuzz test and fix them together, along with a checkin that enables clear for the fuzz tests.

didn't realize you opened #16340 as well. So ignore a bit of the above. After checking that in, you should be able to start turning on the fuzz tests as part of this PR.

**Comment by clarenceli-msft** [comment]
> > It's a bit hard to judge this in isolation, as I don't think our map coverage is particularly good and there are other issues that make fuzz tests fail. Seeing as this change is very small, I'd kind of prefer you investigate the other skipped fuzz test and fix them together, along with a checkin that enables clear for the fuzz tests.
> 
> didn't realize you opened #16340 as well. So ignore a bit of the above. After checking that in, you should be able to start turning on the fuzz tests as part of this PR.

have enabled the fuzz tests with clear operation

**Review by Abe27342 — APPROVED** [review]

**Pull request merged**